### PR TITLE
Replaced with shape theme

### LINKED ISSUE #120: Define shape theming
## Kind (Required)

- Improvement

 ## Overview (Required)

- Right now it is defined for each widget, but it should be possible to set it with theme. 

 ## Links	

- https://github.com/DroidKaigi/conference-app-2021/blob/610ad01d00a673211e9bcef716762c367cdb53d6/uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/theme/Shape.kt#L8

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #120

## Overview (Required)
- Define shape theme and replaced using shape theme in some components.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/theme/Shape.kt [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 
 val shapes = Shapes(
-    small = RoundedCornerShape(4.dp),
-    medium = RoundedCornerShape(4.dp),
-    large = RoundedCornerShape(0.dp)
+    small = CutCornerShape(
+        topStart = 8.dp,
+        bottomEnd = 8.dp
+    ),
+    medium = CutCornerShape(
```
Comment (NUmeroAndDev) [inline uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/theme/Shape.kt:12]: There was no medium components(ex. Card, Dialog) in Figma, so temporarily made the same as small shape.

**Comment by takahirom** [comment]
👀

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedItem.kt [hunk 2]
```diff
@@ -54,10 +53,7 @@ fun FeedItem(
                     }
                     .background(
                         color = feedItem.media.color(),
-                        shape = CutCornerShape(
-                            topStart = 8.dp,
-                            bottomEnd = 8.dp
-                        )
+                        shape = MaterialTheme.shapes.small
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedItem.kt:56]: I didn't know that shape could be used this way! Thank you!

**Comment by takahirom** [comment]
LGTM! Thanks for first contribute 🥇

**Pull request merged**